### PR TITLE
css-two-column-layout Nathan Chon

### PR DESCRIPTION
**Pull request by Nathan-Chon**
Screenshot_20230126_051808

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+.column-one {
+  flex-basis: 100%;
+  text-align: center;
+  max-width: 800px;
+  margin: 50px;
+}
+
+.column-two {
```
Comment (robgmerrill) [inline css-two-column-layout/styles.css:19]: I would rename the class `.column-two`. Something like `.column-one-half`,  `.column-half`, '.col-half`.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! Couple of minor suggestions on my part.

**Comment by Nathan-Chon** [comment]
Screenshot_20230126_051808
Revised version

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**